NuRadioReco/modules/ARA/triggerSimulator.py

```diff
@@ -200,49 +247,46 @@ def run(self, evt, station, det,
 
         # No coincidence requirement yet
         trigger = {}
+        trigger_times = []
         for channel in station.iter_channels():
             channel_id = channel.get_id()
             if channel_id not in triggered_channels:
                 continue
             trigger[channel_id] = self.has_triggered(channel)
+            if trigger[channel_id]:
+                times = channel.get_times()
+                trace_after_diode = self.tunnel_diode(channel)
+                arg_trigger = np.argwhere( trace_after_diode == np.min(trace_after_diode) )[0,0]
```

Comment:
Isn't this equivalent to just writing arg_trigger=np.argmin(trace_after_diode)?

Reply:
You're right. I've changed it.